src/pudl/metadata/classes.py

```diff
@@ -933,9 +933,24 @@ def raw_datapackage_title(self) -> str:
         """Construct a datapackage title for the raw data source."""
         return f"PUDL Raw f{self.title}"
 
-    def to_raw_datapackage_json(self) -> pydantic.Json:
+    def raw_datapackage_sources(self) -> List[Dict[str, str]]:
+        """Construct datapackage sources for raw data source."""
+        return [{"title": self.title, "name": self.path}]
```

Comment:
In the [datapackage specification](https://specs.frictionlessdata.io/data-package/#sources), it looks like in the individual `source` elements what's called `name` here should probably be called `path`. Does that seem right?

Reply:
Yeah I think that's correct. Fixed now